### PR TITLE
[presentation-api] fix typos

### PR DESCRIPTION
**Pull request by tomoyukilabs**
This PR fixes several typos in one of receiving UA tests.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-16 10:17:06
Finished: 2017-09-16 10:22:39


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18597)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/276215181)

File: presentation-api/receiving-ua/support/PresentationConnection_onmessage_receiving-ua.html [hunk 1]
```diff
@@ -63,8 +63,8 @@
 
     assert_equals(connection.binaryType, 'arraybuffer', 'the default value of binaryType is "arraybuffer"');
     const eventWatcher = new EventWatcher(t, connection, 'message');
-    return eventWatcher.wait_for('message').then(evt => {
-      checkEvent(evt);
```
Comment (Honry) [inline presentation-api/receiving-ua/support/PresentationConnection_onmessage_receiving-ua.html]: @tomoyukilabs, I don't think it's typo, we often use `evt` as abbreviation of `event`.

Comment (tomoyukilabs) [reply presentation-api/receiving-ua/support/PresentationConnection_onmessage_receiving-ua.html]: Yes, it’s correct. However, the next line contains `assert_equals(event.data, ...)`, which causes an error. So, this PR is intended to fix this inconsistency.

Comment (Honry) [reply presentation-api/receiving-ua/support/PresentationConnection_onmessage_receiving-ua.html]: Oh, sorry, you are right.

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**